cypress/utils/dashboards/data_explorer/elements.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const DATA_EXPLORER_PAGE_ELEMENTS = {
+  NEW_SEARCH_BUTTON: 'discoverNewButton',
+  DISCOVER_QUERY_HITS: 'discoverQueryHits',
+  DATASET_SELECTOR_BUTTON: 'datasetSelectorButton',
+  ALL_DATASETS_BUTTON: 'datasetSelectorAdvancedButton',
+  DATASET_EXPLORER_WINDOW: 'datasetExplorerWindow',
+  DATASET_SELECTOR_NEXT_BUTTON: 'datasetSelectorNext',
+  DATASET_SELECTOR_LANGUAGE_SELECTOR: 'advancedSelectorLanguageSelect',
+  DATASET_SELECTOR_TIME_SELECTOR: 'advancedSelectorTimeFieldSelect',
+  DATASET_SELECTOR_SELECT_DATA_BUTTON: 'advancedSelectorConfirmButton',
+  DOC_TABLE: 'docTable',
+  DOC_TABLE_ROW_FIELD: 'docTableField',
+  DOC_TABLE_EXPAND_TOGGLE_COLUMN_BUTTON: 'docTableExpandToggleColumn',
+  DOC_TABLE_EXPANDED_DOC_COLUMN_ROW_PREFIX: 'tableDocViewRow-',
+  DOC_TABLE_EXPANDED_DOC_TOGGLE_COLUMN_BUTTON: 'toggleColumnButton',
+  DOC_TABLE_EXPANDED_DOC_COLUMN_ADD_INCLUSIVE_FILTER_BUTTON: 'addInclusiveFilterButton',
+  DOC_TABLE_EXPANDED_DOC_COLUMN_REMOVE_INCLUSIVE_FILTER_BUTTON: 'removeInclusiveFilterButton',
+  DOC_TABLE_EXPANDED_DOC_COLUMN_EXISTS_FILTER_BUTTON: 'addExistsFilterButton',
+  DOC_TABLE_HEADER_FIELD_PREFIX: 'docTableHeader-',
+  TABLE_FIELD_FILTER_FOR_BUTTON: 'filterForValue',
+  TABLE_FIELD_FILTER_OUT_BUTTON: 'filterOutValue',
+  SEARCH_DATE_PICKER_BUTTON: 'superDatePickerShowDatesButton',
+  SEARCH_DATE_PICKER_START_DATE_BUTTON: 'superDatePickerstartDatePopoverButton',
+  SEARCH_DATE_PICKER_END_DATE_BUTTON: 'superDatePickerendDatePopoverButton',
+  SEARCH_DATE_PICKER_RELATIVE_TAB: 'superDatePickerRelativeTab',
+  SEARCH_DATE_RELATIVE_PICKER_INPUT: 'superDatePickerRelativeDateInputNumber',
```

Comment:
should remove these relative picker constants.
for absolute ones, they are only used and wrapped in `setTopNavDate` method. so no other places will use them, we don't need constants for them.